src/Uno.Sdk/packages.json

```diff
@@ -147,6 +147,15 @@
 			"Prism.Uno.WinUI.Markup"
 		]
 	},
+	{
+		"group": "UnoFonts",
+		"version": "2.4.0-dev.6",
+		"packages": [
+			"Uno.Fonts.OpenSans",
+			"Uno.Fonts.Fluent",
+			"Uno.Fonts.Roboto"
+		]
+	},
```

Comment:
I'm not really sure if this is the way to go. Currently, the SDK can only include `Uno.Fonts.OpenSans` through `OpenSans` feature. So I'm not really sure if we should group these together. They are always published together and have the same version, but I'm not sure if this is enough motivation to group them.

cc @dansiegel

Reply:
The same packaged version is good enough for now to group them.